src/components/Product.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Product = ({ label, imageSrc, name, price }) => {
+    return (
+        <article>
+            <span className="label">{label}</span>
+            <img src={imageSrc} alt={name} />
+            <p>{name}</p>
+            <h4>{price}</h4>
```

Comment:
Je zou het euro teken hier al voor kunnen zetten. Hoef je alleen het getal (prijs) in te voeren. (Als je tassen buiten Europa gaat verkopen is jouw oplossing weer handiger ;) )